db/migrate/20210302002326_create_thumbs.rb

```diff
@@ -0,0 +1,23 @@
+class CreateThumbs < ActiveRecord::Migration[6.1]
+  def change
+    create_table :thumbs do |t|
+      t.references :thumber, null: false, foreign_key: { to_table: :users }
+      t.text :body
+      t.references :user, null: false, foreign_key: true
+      t.integer :thumbcount
+      t.string :thumbable_type
+
+      t.timestamps
+    end
+    rename_column :thumbs, :user_id, :thumbable_id
+
+    reversible do |dir|
+      dir.up { Thumb.update_all("Thumbable_type='User'") }
+      dir.down { Thumb.where.not(thumbable_type: 'User').delete_all }
+    end
+    add_index :thumbs, %i[thumbable_type thumbable_id]
+    if foreign_key_exists?(:thumbs, column: :thumbable_id)
+      remove_foreign_key :thumbs, column: :thumbable_id
+    end
```

Comment:
Is this just copied from the comment migrations? There's a lot of code that shouldn't be here (it's only there because comments were incrementally designed) and fields that don't seem to be relevant to the model.